src/burn/engine/core.cpp

```diff
@@ -1508,8 +1508,8 @@ static void LogPackages(
             LogId(REPORT_STANDARD, MSG_PLANNED_PACKAGE, pPackage->sczId, LoggingPackageStateToString(pPackage->currentState), LoggingRequestStateToString(pPackage->defaultRequested), LoggingRequestStateToString(pPackage->requested), LoggingActionStateToString(pPackage->execute), LoggingActionStateToString(pPackage->rollback), LoggingBoolToString(pPackage->fAcquire), LoggingBoolToString(pPackage->fUncache), LoggingDependencyActionToString(pPackage->dependencyExecute));
         }
 
-        // Display related bundles last if installing, modifying, or repairing.
-        if (BOOTSTRAPPER_ACTION_UNINSTALL < action && 0 < pRelatedBundles->cRelatedBundles)
+        // Display related bundles last if caching, installing, modifying, or repairing.
+        if ((BOOTSTRAPPER_ACTION_UNINSTALL < action || BOOTSTRAPPER_ACTION_CACHE == action) && 0 < pRelatedBundles->cRelatedBundles)
```

Comment:
But from his comment on his PR, he added support by allowing a BA to mark all packages for Cache instead of, for example, Install. Or did I misunderstand what his PR was doing?